patches/dompurify+2.5.8.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/node_modules/dompurify/dist/purify.cjs.js b/node_modules/dompurify/dist/purify.cjs.js
```

Comment:
Patched compared to https://github.com/cure53/DOMPurify/commit/d18ffcb554e0001748865da03ac75dd7829f0f02#diff-8e647ca2a1d9380f03114f7df5e6a70563c9cb40b03dca353a43ce84b974c2cdR6